content/scout/quickstart.md

```diff
@@ -19,6 +16,10 @@ This guide takes a vulnerable container image and shows you how to use Docker
 Scout to identify and fix the vulnerabilities, compare image versions over time,
 and share the results with your team.
 
+_The following video shows an end-to-end workflow of using Docker Scout to remediate a reported vulnerability_.
+
+<iframe class="border-0 w-full aspect-video mb-8" allow="fullscreen" src="https://www.loom.com/embed/e066986569924555a2546139f5f61349?sid=6e29be62-78ba-4aa7-a1f6-15f96c37d916"></iframe>
+
 ## Setup
```

Comment:
In other areas of docs, we've added steps in the headers. e.g 'Step 1: Setup' etc. May be useful here too and worth considering for consistency?

https://docs.docker.com/admin/organization/security-settings/sso-configuration/
https://docs.docker.com/admin/organization/onboard/
https://docs.docker.com/compose/gettingstarted/